### ISSUE TITLE
Relicense VS:UtCS under GPLv3+ ?

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Assets-Masters#34]
See also https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/548

**Comment by stephengtuggy** [comment]
I believe that the VS:UtCS assets are currently under GPLv2+, for the most part, with some exceptions e.g. for the music. Not sure how many of the text-format files actually list a copyleft notice at the top, though. We may need to add that in a bunch of places.

**Comment by klaussfreire** [comment]
Tracking all authors can take a while. For my part, I would have no problem relicensing under GPLv3+

Music, however, I think is mostly some form of CC. Licensing music in GPLv3 would require distributing sources, which we don't have.

**Comment by stephengtuggy** [comment]
@klaussfreire Makes sense. So I guess we can leave the music under its current license(s), excluding them from any changes we make to other parts of the project.

**Comment by BenjamenMeyer** [comment]
@stephengtuggy Artwork will likely fall into the same issue as music is my guess.

@klaussfreire I'm not saying we need to track the authors down; but as a first step we need to review to see if we do.

We should also make the licensing clearer using the different licenses as appropriate:
- GPLv2+/v3+ should only be code
- CC (or some form of it) should be for any artwork (images, music, etc)
- FDL (https://www.gnu.org/licenses/fdl-1.3.en.html) should be for documentation if we don't want to use CC there too.
